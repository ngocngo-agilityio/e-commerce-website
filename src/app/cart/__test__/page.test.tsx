// Libs
import { render } from '@testing-library/react';

// Components
import Cart from '../page';

describe('Cart page', () => {
  test('should render Cart page successfully', () => {
    const { getByText } = render(<Cart />);

    expect(getByText('Home')).toBeInTheDocument();
  });

  test('should match snapshot for Cart page', () => {
    const { container } = render(<Cart />);

    expect(container).toMatchSnapshot();
  });
});
