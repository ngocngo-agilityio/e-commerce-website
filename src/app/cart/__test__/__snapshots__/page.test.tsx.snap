// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart page should match snapshot for Cart page 1`] = `
<div>
  <div
    class="chakra-container css-1x3h7hf"
  >
    <nav
      aria-label="breadcrumb"
      class="chakra-breadcrumb css-7wlyfr"
    >
      <ol
        class="chakra-breadcrumb__list css-70qvj9"
      >
        <li
          class="chakra-breadcrumb__list-item css-18biwo"
        >
          <a
            href="/"
          >
            Home
          </a>
          <span
            class="css-t4q1nq"
            role="presentation"
          >
            /
          </span>
        </li>
        <li
          class="chakra-breadcrumb__list-item css-plisa0"
        >
          <a
            href="/cart"
          >
            Cart
          </a>
        </li>
      </ol>
    </nav>
    <table
      class="chakra-table css-0"
    >
      <thead
        class="css-0"
      >
        <tr
          class="css-0"
        >
          <th
            class="css-0"
          />
          <th
            class="css-0"
          />
          <th
            class="css-0"
          >
            Product
          </th>
          <th
            class="css-0"
          >
            Price
          </th>
          <th
            class="css-0"
          >
            Quantity
          </th>
          <th
            class="css-0"
          >
            Total
          </th>
        </tr>
      </thead>
      <tbody
        class="css-0"
      >
        <tr
          class="css-0"
        >
          <td
            class="css-rtu7mb"
            colspan="6"
          >
            No products in your cart.
          </td>
        </tr>
      </tbody>
    </table>
    <h2
      class="chakra-heading css-1k51fm5"
    >
      Cart Totals
    </h2>
    <div
      class="css-3u1iq8"
    >
      <div
        class="css-19y425s"
      >
        <p
          class="chakra-text css-0"
        >
          Subtotal
        </p>
        <p
          class="chakra-text css-0"
        >
          $0.00
        </p>
      </div>
      <hr
        aria-orientation="horizontal"
        class="chakra-divider css-1upb9tn"
      />
      <div
        class="css-19y425s"
      >
        <p
          class="chakra-text css-0"
        >
          ShippingFree
        </p>
        <p
          class="chakra-text css-0"
        >
          FREE!!!
        </p>
      </div>
      <hr
        aria-orientation="horizontal"
        class="chakra-divider css-1upb9tn"
      />
      <div
        class="css-19y425s"
      >
        <p
          class="chakra-text css-19q2mny"
        >
          Total
        </p>
        <p
          class="chakra-text css-0"
        >
          $0.00
        </p>
      </div>
    </div>
    <button
      class="chakra-button css-70vipg"
      type="button"
    >
      Proceed to Checkout
    </button>
  </div>
</div>
`;
